state-manager/Dockerfile

```
@@ -1,14 +1,34 @@
 FROM python:3.12-slim-bookworm
-COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+
+# Install curl for downloading uv
+RUN apt-get update && apt-get install -y curl ca-certificates \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)